src/test/scala/com/stratio/hermes/utils/HermesTest.scala

```diff
@@ -135,6 +135,33 @@ class HermesTest extends FlatSpec with Matchers {
     //scalastyle:on
   }
 
+  it should "generate valid locations: ES and US locales" in {
+
+    val hermesES = Hermes("ES")
+    hermesES.Geo.geoModel should contain (hermesES.Geo geolocation)
```

Comment:
It doesn't matter but to be consistent with the next check I'd add a period after the Geo -> (hermesEs.Geo.geolocation)